umbraco/Umbraco-CMS
# Accessibility Fixes for Users and Packages pages

### Prerequisites

This is related to the Accessibility issue here: https://github.com/umbraco/Umbraco-CMS/issues/12546

If there's an existing issue for this PR then this:
- Fixes umbraco/Umbraco-CMS.Accessibility.Issues#51
- Fixes umbraco/Umbraco-CMS.Accessibility.Issues#69
- Fixes umbraco/Umbraco-CMS.Accessibility.Issues#68

### Description
**Fixes included in this PR:** 

**1. Fixes umbraco/[Umbraco-CMS.Accessibility.Issues#51](https://github.com/umbraco/Umbraco-CMS.Accessibility.Issues/issues/51) (Packages - Skipped heading level(s))**

**What did you do and why ?**
- Updated the h4 headings to be h2 to follow the heading hierarchy and applied styling to these so they matched the current headings

**How to test ?**
- Go to the Packages page, and run the [Accessibility Insights - Chrome Extension](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni) to check that no errors are shown for skipped headings

**2. Fixes umbraco/[Umbraco-CMS.Accessibility.Issues#69 ](https://github.com/umbraco/Umbraco-CMS.Accessibility.Issues/issues/69) (Add User - Skipped heading level(s))**

**What did you do and why ?**
- Updated the h3 headings to be h2 to follow the heading hierarchy and applied styling to these so they matched the current headings

**How to test ?**
- Go to the Users - Create User page, and run the [Accessibility Insights - Chrome Extension](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni) to check that no errors are shown for skipped headings

**3. Fixes umbraco/[Umbraco-CMS.Accessibility.Issues#68](https://github.com/umbraco/Umbraco-CMS.Accessibility.Issues/issues/68) (Add User - Missing form labels)**

What did you do and why ?
- Updated code to apply the correct "for" identifier to point to the form field and add id to form field to link the label and form fields together

How to test ?
- Go to the User - Create User page, and run the [Accessibility Insights - Chrome Extension](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni) to check that no errors are shown for missing form labels
